### ISSUE TITLE
Comma-dangle: only-multiline does not report dangling comma after rest parameters

### ISSUE THREAD

**Issue by salik1992** [babel/babel-eslint#804]
**Environment**
babel-eslint: v8.2.6
ESLint version: v6.5.1
Node version: v12.11.1
npm version: v6.11.3

**ESLint config**
```js
module.exports = {
    parser: 'babel-eslint',
    extends: [
        'airbnb',
    ],
    env: {
        node: true,
        browser: true,
        es6: true,
        jest: true,
        mocha: true
    },
    parserOptions: {
        ecmaFeatures: {
            experimentalObjectRestSpread: true
        }
    },
    plugins: [
        'jest',
        'detox',
        'mocha'
    ],
    rules: {
        'arrow-body-style': ['warn', 'as-needed'],
        camelcase: 'off',
        'class-methods-use-this': 'off',
        indent: ['error', 4],
        'max-statements': ['warn', 20],
        'new-cap': ['error', { capIsNew: false }],
        'no-duplicate-imports': 'off',
        'no-param-reassign': ['error', { ignorePropertyModificationsFor: ['el', 'ref', 'event'] }],
        'no-plusplus': 'off',
        'no-shadow': ['error', { builtinGlobals: false, hoist: 'functions', allow: ['describe'] }],
        'react/jsx-filename-extension': ['error', { extensions: ['.js'] }],
        'react/jsx-indent': ['error', 4],
        'react/jsx-indent-props': ['error', 4],
        'react/require-extension': 'off',
        'react/sort-comp': [
            'error',
            {
                order: [
                    'type-annotations',
                    'static-methods',
                    'lifecycle',
                    '/^on.+$/',
                    '/^(get|set)(?!(InitialState$|DefaultProps$|ChildContext$)).+$/',
                    'everything-else',
                    '/^render.+$/',
                    'render'
                ]
            }
        ],
        'object-curly-newline': ['error', {
            ObjectExpression: { minProperties: 6, multiline: true, consistent: true },
            ObjectPattern: { minProperties: 6, multiline: true, consistent: true }
        }],

        'import/prefer-default-export': 'off',
        'import/no-extraneous-dependencies': 'off',
        'import/no-duplicates': 'error',
        'import/no-named-as-default': 'off',

        'jsx-a11y/no-static-element-interactions': 'off',
        'jsx-a11y/no-noninteractive-tabindex': 'off',
        'jsx-a11y/media-has-caption': 'off',
        'jsx-a11y/mouse-events-have-key-events': 'off',

        'comma-dangle': ['error', 'only-multiline'],

        'mocha/no-exclusive-tests': 'error'
    },
};
```

**Code example**
```js
const Container = ({
    width,
    height,
    ...props,
}) => (
    // JSX body
);
```

**Expected to happen**
The comma after `...props` should be handled as an error `Unexpected trailing comma` as it does for `always-multiline` setting.

**What happened**
With `only-multiline` setting the code passes without an error.

Originally the issue has been opened in eslint repository (https://github.com/eslint/eslint/issues/12435), but they say it should be in this repository.

**Comment by nicolo-ribaudo** [comment]
Since that is a syntax error, it's a Bug with `@babel/parser`.

The good news is that it has been fixed a few days ago in v7.6.4 😛
https://github.com/babel/babel/commit/34937f13d52f4dffcf74d10b10ac96cbeadb987d

**Comment by kaicataldo** [comment]
Thank you for the PR. Now that [`@babel/eslint-parser` has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
